packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -115,6 +115,7 @@ export function useFocusManager(): FocusManager {
   return useContext(FocusContext);
 }
 
+/** @private */
```

Comment:
These don't need to be documented or marked as private since they aren't exported.

Reply:
yeah, i can remove them, i was trying to figure out a way to cut down on the noise when i flip on require jsdocs. the better way would be to get the correct AST selector, but I was having a tough time of it.
I feel like it should be something along the lines of
`'ExportNamedDeclaration TSInterfaceDeclaration TSPropertySignature', 'ExportNamedDeclaration TSInterfaceDeclaration TSMethodSignature'`
but it wasn't quite working, I'll remove the @private's though and maybe we can figure out those selectors later